LambaAnalyzer/src/LambdaAnalyzer.cc

```diff
@@ -1161,6 +1165,33 @@ void LambdaAnalyzer::loop(const edm::Event& iEvent, const edm::EventSetup& iSetu
         	//genLambdaVtx->position is the lambda->pion,proton vertex.  genLambda.v() is the vertex the lambda is created at
 			//std::cout << "Calculating Gen Lambda Flight Length" << std::endl;
         	GenFlightLength.push_back(sqrt(pow(genLambdaVtx->position().x()-genLambda.vx(),2) + pow(genLambdaVtx->position().y() - genLambda.vy(),2) + pow(genLambdaVtx->position().z() - genLambda.vz(),2)));
+            
+            // Gen Protons and Pions:
+            GenProtonPt.push_back(genProton.pt());
```

Comment:
This is a bit picky, but could you fix the spacing?  Some tabs are goofing it up.  I propose you replace each tab with 4 spaces.

Reply:
Should be fixed now!